ios/Podfile

```diff
@@ -1,9 +1,11 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+require_relative '../node_modules/react-native-unimodules/cocoapods.rb'
 
-platform :ios, '10.0'
+platform :ios, '11.0'
```

Comment:
Wait! This is the same issue we ran into with that other camera library. We can't bump to iOS 11 until we get clearance from RJ.